src/core.jl

```diff
@@ -100,7 +110,15 @@ function _test_reference(equiv, rendermode, file::File, actual::T) where T
         return nothing # skip current test case
     end
 
+    # file exists
+    actual = preprocess(file, actual; kw...)
```

Comment:
Can we not reused the `actual` variable?

Maybe before this line (in the function Sig) it should be `raw_actual` or something

Reply:
done in https://github.com/Evizero/ReferenceTests.jl/pull/37/commits/8bb84a23c3776995332c7cf6091ce1bb1d53b5d9